tests/e2e/test_api.py

```diff
@@ -85,21 +86,29 @@ def install_ols() -> tuple[str, str, str]:
     metrics_token = cluster_utils.get_token_for("metrics-test-user")
     print("created test service account users")
 
-    # install the operator catalog
-    cluster_utils.run_oc(
-        ["create", "-f", "tests/config/operator_install/catalog.yaml"],
-        ignore_existing_resource=True,
-    )
-    cluster_utils.run_oc(
-        ["create", "-f", "tests/config/operator_install/operatorgroup.yaml"],
-        ignore_existing_resource=True,
-    )
-    cluster_utils.run_oc(
-        ["create", "-f", "tests/config/operator_install/subscription.yaml"],
-        ignore_existing_resource=True,
-    )
-
-    print("Created catalog+subscription")
+    # install the operator from bundle
+    print("Installing OLS operator from bundle")
+    try:
+        subprocess.run(  # noqa: S603
+            [  # noqa: S607
+                "operator-sdk",
+                "run",
+                "bundle",
+                "--timeout=10m",
```

Comment:
we can try increase it to 20 minutes to see if that helps, maybe the CI cluster is slower than normal